e2e/inference/README.md

```diff
@@ -39,7 +39,7 @@ Create `AcceleratorProfile` in the `redhat-ods-applications` namespace
 $ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/e2e/inference/accelerator_profile.yaml
```

Comment:
@vbedida79  Could we split the e2e related contents to another PR?

Reply:
Done. Added in https://github.com/intel/intel-technology-enabling-for-openshift/pull/285